Compiler error

Steps:
1. Clone the repo (commit `c070cf34f195a5fd9dbbbf95533f5b2e350fd01d`)
2. `bazel build @heir//tools:heir-opt`

This produces an error:
```
lib/Transforms/YosysOptimizer/LUTImporter.cpp: In member function 'virtual llvm::SmallVector<Yosys::RTLIL::SigSpec> mlir::heir::LUTImporter::getInputs(Yosys::RTLIL::Cell*) const':
lib/Transforms/YosysOptimizer/LUTImporter.cpp:62:10: error: could not convert 'reversed' from 'SmallVector<[...],4>' to 'SmallVector<[...],1>'
   62 |   return reversed;
      |          ^~~~~~~~
      |          |
      |          SmallVector<[...],4>
Target //tools:heir-opt failed to build
```

--- a/lib/Transforms/YosysOptimizer/BooleanGateImporter.cpp
+++ b/lib/Transforms/YosysOptimizer/BooleanGateImporter.cpp
@@ -40,7 +40,7 @@ mlir::Operation *BooleanGateImporter::createOp(Yosys::RTLIL::Cell *cell,
 SmallVector<Yosys::RTLIL::SigSpec> BooleanGateImporter::getInputs(
     Yosys::RTLIL::Cell *cell) const {
   // Return all non-Y named attributes.
-  SmallVector<Yosys::RTLIL::SigSpec, 4> inputs;
+  SmallVector<Yosys::RTLIL::SigSpec> inputs;
   for (auto &conn : cell->connections()) {
     if (conn.first.contains("Y")) {
       continue;

--- a/lib/Transforms/YosysOptimizer/LUTImporter.cpp
+++ b/lib/Transforms/YosysOptimizer/LUTImporter.cpp
@@ -54,7 +54,7 @@ SmallVector<Yosys::RTLIL::SigSpec> LUTImporter::getInputs(
   }
   // Alphabetical order gives LSB to MSB, but LUT operations order their inputs
   // from MSB to LSB.
-  SmallVector<Yosys::RTLIL::SigSpec, 4> reversed;
+  SmallVector<Yosys::RTLIL::SigSpec> reversed;
   reversed.reserve(inputs.size());
   for (unsigned i = 0; i < inputs.size(); i++) {
     reversed.push_back(inputs[inputs.size() - i - 1]);

